workflow/rocoto/gefs_tasks.py

```diff
@@ -483,3 +483,45 @@ def wavepostpnt(self):
         task = rocoto.create_task(member_metatask_dict)
 
         return task
+
+    def arch(self):
+        deps = []
+        dep_dict = {'type': 'metatask', 'name': f'atmos_prod'}
```

Comment:
```suggestion
        dep_dict = {'type': 'metatask', 'name': 'atmos_prod'}
```
`f` is used when there is something to substitute.  It is unnecessary here and in the rest of the dependencies.

Reply:
fixed